kadai-adapter-camunda-wildfly-example/src/main/java/io/kadai/adapter/KadaiAdapterWildFlyApplication.java

```diff
@@ -0,0 +1,37 @@
+package io.kadai.adapter;
+
+import io.kadai.adapter.configuration.AdapterConfiguration;
+import io.kadai.adapter.kadaiconnector.config.KadaiSystemConnectorConfiguration;
+import io.kadai.adapter.systemconnector.camunda.config.CamundaSystemConnectorConfiguration;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Import;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+/**
+ * Example Application showing the implementation of kadai-adapter for jboss application server.
+ */
+@SpringBootApplication
+@EnableScheduling
+@ComponentScan(
+    basePackages = {
+      "io.kadai.adapter",
+      "io.kadai.adapter.configuration",
+      "io.kadai",
+      "io.kadai.adapter.systemconnector.camunda.config",
+      "io.kadai.adapter.kadaiconnector.config"
+    })
+@SuppressWarnings("checkstyle:Indentation")
```

Comment:
But why? Can't we just format the file properly?

Reply:
Let's just leave it like that for the moment